tests/robot-tests/tests/libs/create_emulator_release_files.py

```diff
@@ -162,37 +166,60 @@ def create_public_release_files(self):
             container_client = self.blob_service_client.create_container(
                 "downloads")
         except ResourceExistsError:
+            pass
+
```

Comment:
I think this change can be reverted to what it was originally. It was trying to create the blob service, and if that fails, it was then getting it.

After your changes, it tries to create the blob service, and if it fails, it passes, which means we now don't have the container_client. 

I think maybe you've not realised that both the `pass` and `container_client = self.blob_service_client.get_container_client(` are inside the same `except` scope.

If you did have `container_client = self.blob_service_client.get_container_client(` outside the `except` scope, then you would potentially be getting `container_client` twice, when the original code would only ever get it once.